README.md

```diff
@@ -49,22 +49,59 @@ Here we list formats that have been used or proposed thus far in the community (
 
 ## Standard Matrices
 
-We benchmark data formats and APIs on standard matrices. These matrices are found at the following S3 bucket.
-
-[TODO add S3 bucket details]
-
-[TODO describe matrices below]
+We benchmark data formats and APIs on standard matrices. These matrices are found in the public S3 bucket `matrix-format-test-data`.
+
+That bucket has two relevant folders, `matrices` and `sources`. The `sources` folder contains raw data from a number of different data sources, in the form that they were originally delivered. These are meant to span a range of matrix sizes and properties so we can develop tests for diverse use cases:
+
+| Source Name | Description |
+| ----------- | ----------- |
+| tenx_mouse_neuron_1M | The 1.3 million mouse neuron dataset released by 10x. |
+| tenx_mouse_neuron_20k | A sample of 20k cells from the 1.3 million cell dataset, also provided by 10x. |
+| immune_cell_census_cord_blood | Cord blood cells from the Immune Cell Atlas dataset released for the HCA preview. 384,000 cells sequenced with 10x.|
+| immune_cell_census_bone_marrow | Bone marrow cells from the Immune Cell Atlas dataset released for the HCA preview. 378,000 cells sequenced with 10x. |
+| GSE84465_whole | 3589 cells sequenced with SmartSeq2 from GEO Series GSE84465. |
+| GSE84465_split | 3589 cells sequenced with SmartSeq2 from GEO Series GSE84465, split so there is one matrix file per cell. |
+
+The `matrices` folder contains two levels of subfolders. The first is the matrix format. These are the formats that are being tested for performance among a set of different tasks.
+
+| Format Name | Description |
+| ----------- | ----------- |
```

Comment:
I also suggest to add [netCDF](http://xarray.pydata.org/en/stable/io.html#netcdf) format to this list. It's the default format for xarray and it will make it much easier to standardize the column and row annotations.

Again, this is outside of this PR's scope (I am working on a simple code for converting csv to netCDF and can help with this).

Reply:
I'm interested in more details about the interop between netCDF and HDF5, if you have any. 

My working/optimistic assumption is that netCDF libraries can be used to read HDF5 files in all cases we care about; this was tested a bit by @tomwhite at [tomwhite/hdf5-java-cloud](https://github.com/tomwhite/hdf5-java-cloud), which is still the only way I've seen in any language to read or write HDF5 files directly from/to cloud stores.

I'm looking forward to seeing if xarray's python netCDF support allows this, per discussion on the call yesterday with @ambrosejcarr as well.

The answer to those "level of interop w HDF5" questions will have some bearing on whether we treat/test netCDF as a distinct set of formats from HDF5, I think?

Reply:
I thought netCDF files were essentially restricted HDF5 files and thus interoperability, at least in one direction, should be complete. I don't have a lot of confidence in that belief though.

We can definitely add it into the mix though, I'll make an issue.

Reply:
Yea, [according to wikipedia](https://en.wikipedia.org/wiki/NetCDF):

> The netCDF-4/HDF5 format was introduced in version 4.0; it is the HDF5 data format, with some restrictions.

[This NetCDF FAQ lists the restrictions](https://www.unidata.ucar.edu/software/netcdf/docs/faq.html#How-can-I-convert-HDF5-files-into-netCDF-4-files).

afaik none of the SC HDF5 formats use these features, so I *think* we can optimistically say we are already supporting NetCDF

Reply:
The main reason I mentioned netCDF was because of [xarray](https://github.com/pydata/xarray) which seems to be an important player in the field, according to [Pangeo](http://pangeo-data.org/packages.html#pangeo-core-packages):
>Xarray adopts the [Common Data Model](https://www.unidata.ucar.edu/software/thredds/current/netcdf-java/CDM/) for self- describing scientific data in widespread use in the Earth sciences

I am pretty sure we can still use xarray without having netCDF as one of the formats. So I guess my question is do we want to use xarray as one the core packages in our implementations?